app/services/collators/outgoings_collator.rb

```diff
@@ -53,19 +54,26 @@ def call(submission_date:, person:, gross_income_summary:,
 
         pension_contribution = Calculators::PensionContributionCalculator.call(
           outgoings: outgoings.select { |o| o.instance_of?(Outgoings::PensionContribution) },
-          cash_transactions: gross_income_summary.cash_transactions(:debit, :pension_contribution),
+          cash_transactions: gross_income_summary.cash_transactions.by_operation_and_category(:debit, :pension_contribution),
           regular_transactions: gross_income_summary.regular_transactions.pension_contributions,
           total_gross_income:,
           submission_date:,
         )
 
         council_tax = Calculators::CouncilTaxCalculator.call(
           outgoings: outgoings.select { |o| o.instance_of?(Outgoings::CouncilTax) },
-          cash_transactions: gross_income_summary.cash_transactions(:debit, :council_tax),
+          cash_transactions: gross_income_summary.cash_transactions.by_operation_and_category(:debit, :council_tax),
           regular_transactions: gross_income_summary.regular_transactions.council_tax_payments,
           submission_date:,
         )
 
+        priority_debt_repayment = Calculators::PriorityDebtRepaymentCalculator.call(
+          outgoings: outgoings.select { |o| o.instance_of?(Outgoings::PriorityDebtRepayment) },
+          cash_transactions: gross_income_summary.cash_transactions.by_operation_and_category(:debit, :priority_debt_repayment),
+          regular_transactions: gross_income_summary.regular_transactions.with_operation_and_category("debit", "priority_debt_repayment"),
```

Comment:
could we create a method 'priority_debt_repayments' to go alongside council_tax_payments?